src/mbgl/map/map_context.cpp

```diff
@@ -47,6 +47,14 @@ MapContext::MapContext(uv_loop_t* loop, View& view_, FileSource& fileSource, Map
 }
 
 MapContext::~MapContext() {
+    if (resourceLoader) {
+        // This is likely to be called only on unit tests when
+        // the MapContext is not bound to a Map object.
```

Comment:
Can we avoid code that is specific to unit tests? This sort of defeats the purpose of having unit tests when we special case them in our regular code.